README.md

```diff
@@ -1 +1,17 @@
-# multi-cluster-eval
\ No newline at end of file
+# Multi-cluster LLM Evals
+
+## Currently supported clusters
+- LEONARDO
+- LUMI
+- JURECA
+
+## Usage
+
+1. git clone `https://github.com/OpenEuroLLM/multi-cluster-eval.git`
+2. `source multi-cluster-eval/cluster_env.sh  # sets all relevant environment variables and activates the python environment
+3. Start an interactive session, e.g., `srun -p $default_partition -t $time_limit --gres="gpu:$num_gpu_per_node" --pty /bin/bash>`
+5. Run the eval: `lm_eval --model hf --model_args pretrained=EleutherAI/pythia-160m,revision=step100000,dtype="float," --tasks mmlu --output_path $EVAL_OUTPUT_DIR`
```

Comment:
We should be consistent on how we name our env, right now we have `default_partition` and `EVAL_OUTPUT_DIR` probably makes sense to use upper-case everywhere as its a general bash convention?

Can be done in another step.

```suggestion
4. Run the eval: `lm_eval --model hf --model_args pretrained=EleutherAI/pythia-160m,revision=step100000,dtype="float," --tasks mmlu --output_path $EVAL_OUTPUT_DIR`
```